AdobeStockImageAdminUi/view/adminhtml/web/css/source/_module.less

```diff
@@ -97,8 +97,8 @@
 
             select {
                 border: none;
-                background-image: url(../images/arrows-bg.svg);
-                padding-right: 2.2rem;
+                background-image: url(../images/arrows-bg.svg) !important;
```

Comment:
Can the desired appearance be achieved without `!important` notation?

Reply:
Don't think so, since:
> Added !important to dropdown arrow background as the default background has the !important already ([reference](https://github.com/magento/magento2/blob/2.3-develop/app/design/adminhtml/Magento/backend/web/css/source/forms/_controls.less#L62)) which is overriding our custom dropdown style

Reply:
Hm, that is a defect in my opinion. Would you be able to revert the PR that introduce that commit for magento2 (https://github.com/magento/magento2/pull/25022)? (one that PR will be mereged https://github.com/magento/magento2/issues/24840 should be reopened)

Reply:
Alright, I will do that and then fix this after. Thanks!

Reply:
Great, thank you!